flutter/build.go

```diff
@@ -4,7 +4,12 @@ package flutter
 // Linux Build Tags
 // ----------------
 #cgo linux CFLAGS: -I${SRCDIR}/library
-#cgo linux LDFLAGS: -L${SRCDIR}/library/linux -lflutter_engine -Wl,-rpath,$ORIGIN/flutter/library/linux
+#cgo linux LDFLAGS: -lflutter_engine -Wl,-rpath,flutter/library/linux
 
 */
 import "C"
+
+import (
+	// prevents dep from stripping out the c source files in flutter library.
+	_ "github.com/Drakirus/go-flutter-desktop-embedder/flutter/library"
```

Comment:
i just learned something new.
Very useful!

Reply:
useful in this case being a nasty hack because dep does the wrong thing lol